tests/tests/test_compat.py

```diff
@@ -34,56 +34,80 @@
 container_factory = factory.get_factory()
 
 TIMEOUT = timedelta(minutes=5)
+"""
+ COMPAT_MENDER_VERSIONS array stores the client version we test against.
```

Comment:
Technically is more than the "client" version as it is really a combination of client and addons. I encourage to avoid the "client" keyword as much as possible. For example
```suggestion
 COMPAT_MENDER_VERSIONS array stores the versions of the virtual device images we test against.
```

Reply:
sure. fixed.